### DISCUSSION TITLE
Qiitaの理想をユーザーが気づけない

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#45] [要望・アイデア・不具合]
私は、基本的にサービスは（常識や利用規約の範囲内で）ユーザーが自由に使うべきだと考えます。
範囲外の行為を行った一部のユーザーのために、他のユーザーの今までの使い方を縛るべきではありません。

具体的には「いいねから LGTM への変更」と「質問機能の追加」の 2 つです。

> 流れてきた記事に「いいね」するのではなく、しっかりその内容を読んで、その内容が少なくとも自分にとって良いものだと思ったときに「LGTM（Looks Good To Me）」と言ってもらった方が良いと考えました。

引用: https://blog.qiita.com/like-to-lgtm/

> 記事投稿機能はQiitaの中で
> 「読んだ人も汎用的に扱えるような、ある程度完成された知見を共有する機能」だと考えています。

引用: https://blog.qiita.com/question-release/

どちらも理想が過ぎると感じてしまいました。

別の議論でも話した通り、運営からのお知らせ（Qiita Blog）に辿り着ける人はごくわずかなので、そもそも LGTM や質問機能の意図を知らないユーザーが圧倒的に多いはずです。
知っていたとしても「しっかり読まなきゃ LGTM を押してはいけない」や「汎用的に使える完成された知見じゃなきゃ投稿すべきでない」に納得できるユーザーは多くないと推測します。

私は納得できておらず、少しでもいいと思った記事には読んだ印として LGTM を付けることもあるし、汎用的でも完成されてもいない知見も投稿しています。
それもあり、ストックと質問機能は使ったことがありません。
（運営の理想に反した使い方をしてすみません。ただ世の中のエンジニアの役に立ちたい一心で Qiita に記事を投稿している事実は間違いありません）

どちらも「質のいい記事が増えて適切に評価されてほしい」という想いがあってとのことですが、もう少しユーザーフレンドリーなやり方を模索してもいいと思いました。

**Comment by uhooi** [comment]
えらそうに長文を書いてしまいました…。気に障りましたらすみません。
ただせっかくの機会なので前から思っていることを伝えさせていただきました 🙇

**Reply by murakami** [reply]
確かにそうですね。
読者としては、正式な情報は公式から。ただ、公式の情報では使い方が足りない場合、ちょっとでも情報があると助かりますし、情報が足りない分野は完成度が低くても、複数の投稿があって、それらを比較できただけでも、とても助かっています。

**Reply by uhooi** [reply]
ですね…！
確実に質のいい内容が欲しければ、一次ソースである公式ドキュメントを見ればいいですもんね…。

**Comment by daikisuyama** [comment]
解決策を提示するわけではないので気が引けるのですが、「読んだ人も汎用的に扱えるような、ある程度完成された知見」や「質の良い記事」を適切に評価する指標等がLGTMとは別にあると、個人的には嬉しいです。

また、知見が汎用的であるか及び完成されているかの判断には知見をまとめた上で公開するという経験を一定程度積む必要があると思います。したがって、Qiitaを開放的なコミュニティにするには、「汎用的でも完成されてもいない知見」が書いてある記事も許容されるべきだと思います。

**Comment by kskwtnk** [comment]
@uhooi 

ご意見ありがとうございます！

> えらそうに長文を書いてしまいました…。気に障りましたらすみません。

とんでもないです！このように意見をいただけてとてもありがたいです。

---

まず、以下の前提は全くその通りだと思います

> 私は、基本的にサービスは（常識や利用規約の範囲内で）ユーザーが自由に使うべきだと考えます。
> 範囲外の行為を行った一部のユーザーのために、他のユーザーの今までの使い方を縛るべきではありません。

その上で`理想が過ぎる`についてなのですが、`高い理想に見合うだけの運営活動`と`掲げた理想と現実の乖離`の2つの観点から回答させてください

## 高い理想に見合うだけの運営活動

Qiitaの掲げる理想像は、高い低いの定義は難しいものの、決して低くはないと思います。

これらの理想を掲げるにあたって、私たちが十分な運営活動をできていたかと言えば、NOです。
[以前のブログ](https://blog.qiita.com/qiita-discussions-closed-beta/)でも記載しているような課題感があるため、正直に「できていない」とお答えします。

しかし、今できていないからと理想を取り下げるのも、それはまた違うというスタンスです
サービスとしての理想を体現できるように運営活動を改善していくべきであり、それがユーザーの皆様に対する誠意であると思っています。

今皆様に参加いただいているQiita Discussionsをはじめ、私たちが何を考えて運営しているのか、透明性を上げるつもりでいます。
Blogにたどり着ける人がわずかなのも今起きている事実でありつつ、そもそも「ブログで発信して終わり」のフロー自体も見直すべきだと考えています。

## 掲げた理想と現実の乖離

また、全員が納得できない理想だから掲げるのをやめる、のも違うと思っています。

私たちが理想を掲げていて、全員賛同した上で利用してくださるのがベストですが、いくらか賛同できない箇所があっても「記事を読むのは役に立つ」などの気持ちで使っていただけるのはとても感謝しています。

もちろんuhooiさんの「ストックと質問には賛同できないため機能を使わない」や「気軽なLGTM、完成しきっていない記事」も、理想に反しているからと苦言を呈すようなことはありません。

ただし、理想は運営にとっての要なので簡単に変えるものではないですが、あくまで社会にとって、ユーザーの皆様にとって良いものを提供するために掲げています。
そのため、社会の情勢やユーザーの皆様の価値観も鑑みて自分たちの考え方もブラッシュアップしていくつもりでいます。
そのため今回のような意見は非常にありがたく受け止めています！

---

と、このように「私たちが思う理想」の話ばかりしてしまいましたが

> もう少しユーザーフレンドリーなやり方を模索

今よりも良いやり方は絶対に存在しているため、模索していきます！

こういった意見はなかなか表明するのも難しいと思うのですが、今回Discussionを立てていただいて本当にありがとうございました。
皆様から見たQiitaと、我々が届けたいQiitaが上手く一致するようにこれからも改善していきますので、今後もよろしくお願いします！

**Reply by uhooi** [reply]
このような意見にも丁寧にご回答いただき、ありがとうございます。
そうですね、理想を掲げるのをやめるよりも、理想に近づくためにどう改善するか考えるほうが、運営とユーザーの両方にとって有益だと思いました。
ひとつ訂正ですが、ストックと質問機能は賛同しないからではなく、私にとって不要なので使っていません。（より悪い印象を持たれるかもしれません…。）

Qiita はこれからも読み手と書き手の両方で使わせていただきたいので、こちらこそよろしくお願いいたします！
